bin/fmt.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Applies requisite code formatters to the source tree
+
+set -e
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+source $SCRIPTPATH/use_bazel_go.sh
+
+ROOTDIR=$SCRIPTPATH/..
+cd $ROOTDIR
+
+GO_FILES=$(find adapter cmd model platform proxy test tools/version -type f -name '*.go')
+
+UX=$(uname)
+
+#remove blank lines so gofmt / goimports can do their job
+for fl in ${GO_FILES}; do
+	if [[ ${UX} == "Darwin" ]];then
+		sed -i '' -e "/^import[[:space:]]*(/,/)/{ /^\s*$/d;}" $fl
+	else
+		sed -i -e "/^import[[:space:]]*(/,/)/{ /^\s*$/d;}" $fl
+	fi
+done
+gofmt -s -w ${GO_FILES}
+goimports -w -local istio.io ${GO_FILES}
```

Comment:
isn't goimports a super set of gofmt ? (http://goinbigdata.com/goimports-vs-gofmt/)